util/feature.txt

```diff
@@ -0,0 +1,6 @@
+feature.txt
+feature 1 implementation
+
+feature 2 implementation
+
```

Comment:
Let's add a bit more to the feature 2 implementation. I think it should handle multiple widgets.

Reply:
@essaneid95 see the latest commit for the implementation